v2/go.mod

```diff
@@ -4,7 +4,7 @@ go 1.15
 
 require (
 	github.com/ipfs/go-block-format v0.0.3
-	github.com/ipfs/go-cid v0.0.7
+	github.com/ipfs/go-cid v0.0.8-0.20210702173502-41f2377d9672
```

Comment:
I think you should go ahead and merge https://github.com/ipfs/go-cid/pull/127 and tag a proper 0.0.8 for this.

Reply:
Definitely. Comment made [here](https://protocollabs.slack.com/archives/C01QL4RUA79/p1625500848118200) by Daniel; awaiting response from stewards. There is [an issue](https://github.com/ipld/go-car/issues/91#issuecomment-874215493) in go-car to remind us.